### ISSUE TITLE
Is this still being maintained?

### ISSUE THREAD

**Issue by CarlQLange** [Slava/tern-meteor#39]
Hey @Slava,

Are you still in a position to maintain this project? If not, I'd be happy to get to work updating this for Meteor 1.2.1, with a view to maintaining it in future.
I think this is great work and I'd love to keep it going.

Cheers,
Carl

**Comment by Slava** [comment]
I don't have much time to look into this, but there is a script I used in the past to infer the types from the docs api.

**Comment by Slava** [comment]
If you want to try it yourself, you can run it on the jsdoc output. https://github.com/Slava/tern-meteor/blob/master/jsdocJson2ternjsJson.js

**Comment by zvictor** [comment]
:+1:

**Comment by cvax** [comment]
@CarlQLange were you ever able to get this updated? Thanks a lot for taking the initiative.

**Comment by CarlQLange** [comment]
I was just looking at this again yesterday! I did, but I've since misplaced the things I did. I'll fork it and update it hopefully on Thursday, if that's ok? I'll drop in here to let you know.

**Comment by cvax** [comment]
@CarlQLange that would be perfect! Thanks so much.

**Comment by Slava** [comment]
@CarlQLange I will gladly accept a PR

**Comment by CarlQLange** [comment]
👍 will do

**Comment by CarlQLange** [comment]
OK, well, here's a start. It's a tern typedefs file for Meteor 1.3.3. I haven't exactly tested it though, so there's probably quite a lot of jankiness

https://gist.github.com/8236a34c29ce3149bb38ca1f7dbac83a

**Comment by cvax** [comment]
@CarlQLange Thanks Carl. I just slap it into the meteor.js file replacing the old section right?

**Comment by CarlQLange** [comment]
Yeah! I think so anyway :) The doc links are broken at the minute, I'll try and update with a fixed one soon.

@slava I'll put in a pull request with these updates, and also a readme addition about how to actually get from 0 to a new typedefs file now that the docs and meteor repo have changed pretty significantly, in the next few days. Great job by the way - I wouldn't have been able to do this from scratch, for sure :)

**Comment by Slava** [comment]
@CarlQLange cool! Will be waiting for the PR. Thanks for looking into this.

**Comment by CarlQLange** [comment]
@Slava will do my best to do this over the next few days! Sorry, I've been super busy.

@cvax Did you get this working? How'd it go?

**Comment by cvax** [comment]
@CarlQLange thanks. I believe I got it installed, but I am new to Meteor so couldn't give you a good assessment on the definitions themselves.

**Comment by partounian** [comment]
@CarlQLange @Slava @cvax How's everything going guys? Just want to make sure I didn't miss out on anything.

**Comment by Slava** [comment]
My offer to merge a PR still stands. I am busy with school work and do not
plan to maintain this project besides reviewing prs
2017년 2월 18일 (토) 오후 3:58, Patrick Artounian <notifications@github.com>님이 작성:

**Comment by CarlQLange** [comment]
Sadly I'm busy with work work and can't do much about this at the moment. Sorry!
